src/job-feed/JobCard.js

```diff
@@ -1,73 +1,83 @@
-import React from 'react';
+import React, { useState } from 'react';
 import {
   Button, Card, CardActions, CardContent, Typography,
 } from '@material-ui/core';
 import PropTypes from 'prop-types';
 import './styles.css';
 import { KeyboardArrowRight } from '@material-ui/icons';
+import { Redirect } from 'react-router-dom';
+import { Rating } from '@material-ui/lab';
 
 const JobCard = ({
-  name, description, companyName, jobSkills,
-}) => (
-  <Card className="job-card">
-    <CardContent>
-      <Typography className="job-company-name" variant="h6" gutterBottom>
-        {companyName}
-      </Typography>
-      <hr />
-      <Typography className="job-name" variant="h5" component="h2">
-        {name}
-      </Typography>
-      <Typography className="job-description" color="textSecondary">
-        {description}
-      </Typography>
-      <hr />
-      <Typography color="textSecondary" variant="h6">
-        Job Skills
-      </Typography>
-      <br />
-      {jobSkills?.length > 0 ? jobSkills.map((jobSkill) => (
-        <Typography
-          className="job-skill"
-          key={jobSkill.skill.id}
-          variant="body2"
-          component="p"
-        >
-          {jobSkill.skill.name}
+  id, name, description, companyName, jobSkills,
+}) => {
+  const [buttonClicked, setButtonClicked] = useState(false);
```

Comment:
In a web app, we can have plenty of buttons. You might want to find a more suitable name for this.

Reply:
updated the name to 'jobDetailsButtonClicked'